docs/development.rst

```diff
@@ -78,6 +79,35 @@ Having the order based on a numeric priority in a common directory allows for mo
 
 These items are subject to change. When making changes to the internal functionality, this page needs to be updated as well.
 
+Working with vendored libs
+--------------------------
+
+Vendored libs are external libraries, meaning source code not maintained by bash-it
+developers.
+They are ``git sutrees`` curated in the ``vendor/`` folder. To ease the work with git
```

Comment:
* `subtrees` is missing a `b`.
* `bash-it` should be `Bash-it`.

Reply:
Great, I will get those fixed.

Reply:
done